### ISSUE TITLE
Running tests in windows some times causes timeouts

### ISSUE THREAD

**Issue by karthiknadig** [Microsoft/ptvsd#244]
```
py -3 -m tests tests.highlevel.test_messages.StackTraceTests
running tests...
warning: Debugger speedups using cython not found. Run '"C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python36_64\python.exe" "C:\GIT\ptvsd\ptvsd\pydevd\setup_cython.py" build_ext --inplace' to build.
EC:\GIT\ptvsd\tests\helpers\vsc\_fake.py:167: UserWarning: timed out after 1 seconds waiting for message (<request cmd=initialize seq=0>)
  warnings.warn(msg.format(timeout, handlername))
C:\GIT\ptvsd\tests\helpers\vsc\_fake.py:167: UserWarning: timed out after 1 seconds waiting for message (<request cmd=launch seq=1>)
  warnings.warn(msg.format(timeout, handlername))
C:\GIT\ptvsd\tests\helpers\vsc\_fake.py:167: UserWarning: timed out after 1 seconds waiting for message (<event initialized>)
  warnings.warn(msg.format(timeout, handlername))
C:\GIT\ptvsd\tests\helpers\vsc\_fake.py:167: UserWarning: timed out after 1 seconds waiting for message (<request cmd=configurationDone seq=2>)
  warnings.warn(msg.format(timeout, handlername))
C:\GIT\ptvsd\tests\helpers\vsc\_fake.py:167: UserWarning: timed out after 1 seconds waiting for message (<request cmd=disconnect seq=3>)
  warnings.warn(msg.format(timeout, handlername))
C:\GIT\ptvsd\tests\helpers\vsc\_fake.py:167: UserWarning: timed out after 1 seconds waiting for message (<event exited>)
  warnings.warn(msg.format(timeout, handlername))
C:\GIT\ptvsd\tests\helpers\vsc\_fake.py:167: UserWarning: timed out after 1 seconds waiting for message (<event terminated>)
  warnings.warn(msg.format(timeout, handlername))
```

**Comment by ericsnowcurrently** [comment]
This should be fixed with #283.  #289 and #150 might be related too.

**Comment by karthiknadig** [comment]
@ericsnowcurrently Can confirm this is fixed.
